compute/resource_cluster.go

```diff
@@ -263,6 +262,24 @@ func resourceClusterUpdate(d *schema.ResourceData, m interface{}) error {
 	return resourceClusterRead(d, m)
 }
 
+// wrapClusterRequest helps remove all requests that should not be submitted when instance pool is selected.
+func wrapClusterRequest(clusterModel *Cluster) {
+	// Instance profile id does not exist
+	if reflect.ValueOf(clusterModel.InstancePoolID).IsZero() {
```

Comment:
why do we need to use `reflect` if we can just ` != ""`?

Reply:
changed